path_sequence_planner/CMakeLists.txt

```diff
@@ -13,61 +17,54 @@ else()
   include(${VTK_USE_FILE})
 endif()
 
-find_package(catkin REQUIRED COMPONENTS
-    tool_path_planner
-)
-
 find_package(vtk_viewer REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS
-    include
-  LIBRARIES
-    simple_path_sequence_planner
-  CATKIN_DEPENDS
-    tool_path_planner
-  DEPENDS
-    vtk_viewer
-)
+find_package(tool_path_planner REQUIRED)
 
 include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
+include
```

Comment:
Should get rid of this and make sure include dependencies are only added via `target_include_directories`